### PR TITLE
Update SSR db version tooltip to accurately describe the versioning c…

### LINKED ISSUE #29430: Upgrading from 4.9 or lower version - Cannot update db version to 5.1.0
**Prerequisites (mark completed items with an [x]):**
- [x] I have have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**

If you're upgrading from WooCommerce 4.9 or lower, your DB version stays at 5.0.0 and does not reach 5.1.0

**Expected behavior**

DB version should always match the WooCommerce version:

https://d.pr/i/Q7Bnjt+

**Actual behavior**

If you're upgrading from 5.0.0 to 5.1.0, the db version matches, but if you're upgrading from 4.9 or lower (tested 4.7), the db version stays at 5.0.0 even after a successful DB update notification

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. Upgrade your WooCommerce installation from 4.9.0 to 5.1.0
2. Click on the db update notification 
3. Notice the scheduled actions:
    https://d.pr/i/QhonHw+
4. Check the database version in System Status Report to confirm that it is 5.0.0 even if WooCommerce is 5.1.0
5. Try the database update too from **WooCommerce > Status > Tools > Update database > Click - Update (Button on right)**
6. Notice that there are no actions scheduled for DB update **WooCommerce > Status > Scheduled Actions > Pending**

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
Any. WC Version: 4.9.0 and lower

### PR DESCRIPTION
**Pull request by roykho**
…loses #29430

### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Updating the verbiage for the SSR db version tooltip to more accurately state what is happening.

Closes #29430 

### How to test the changes in this Pull Request:

1. Go to SSR under `Database` section and ensure the tooltip verbiage has been updated.
2.
3.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

* Tweak - Update SSR db version tooltip to more accurately state the db version.

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: includes/admin/views/html-admin-page-status-report.php [hunk 1]
```diff
@@ -514,7 +514,7 @@
 	<tbody>
 		<tr>
 			<td data-export-label="WC Database Version"><?php esc_html_e( 'WooCommerce database version', 'woocommerce' ); ?>:</td>
-			<td class="help"><?php echo wc_help_tip( esc_html__( 'The version of WooCommerce that the database is formatted for. This should be the same as your WooCommerce version.', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
+			<td class="help"><?php echo wc_help_tip( esc_html__( 'The version of WooCommerce that the database is formatted for. Note that it may not match WooCommerce core version and that is normal.', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
```
Comment (vedanshujain) [inline includes/admin/views/html-admin-page-status-report.php:517]: Perhaps we also change 

> The version of WooCommerce that the database is formatted for

to say "Database version for WooCommerce."

**Review by vedanshujain — COMMENTED** [review]
Left a minor suggestion for consideration, looks great otherwise.

**Comment by roykho** [comment]
Updated per suggestion. Another round please.

**Review by vedanshujain — APPROVED** [review]
LGTM!

**Pull request merged**